internal/lib/constants.go

```diff
@@ -174,6 +174,11 @@ const (
 	IPAMProviderCustom                         = "IPAMDNS_TYPE_CUSTOM"
 	SharedVipServiceKey                        = "SharedVipService"
 	HttpRulePkiAndDestCASetErr                 = "PKIProfile and DestinationCA fields are set in the HTTPRule. Only one of the field should be set."
+	IstioCertOutputPath                        = "/etc/istio-output-certs"
+	IstioSecret                                = "istio-secret"
+	IstioPKIProfile                            = "istio-pki"
```

Comment:
Istio PKI profile name is hardcoded, what happens if multiple AKOs are running against teh same controller ?

Reply:
the naming has been updated
the name for istio PKI should now have clustername and namespace name as suffix